lib/utils/eventually-update.js

```diff
@@ -9,7 +9,7 @@ const fse = require('fs-extra');
 const fs = require('fs');
 const fsp = require('fs').promises;
 const spawn = require('child-process-ext/spawn');
-const got = require('got');
+const { default: fetch } = require('node-fetch');
```

Comment:
We're relying on v2 of `node-fetch` and I believe it can be simply required as `const fetch = require('node-fetch');`

Reply:
hi @medikoo, I already tried this but it is not working as expected. it should be imported like `const fetch = require('node-fetch').default;`

Reply:
@debadutta98 it may signal that you have wrong version installed. See that everywhere we use `fetch = require('node-fetch')` -> https://github.com/search?q=repo:serverless/serverless%20node-fetch&type=code

And it's confirmed by tests.

Are tests passing for you? (see https://github.com/serverless/serverless/tree/main/test#readme)

Reply:
yes, @medikoo all test cases are passing except one for  `lib/utils/standalone.js` but I think it is not because of the changes.

Reply:
@debadutta98 in `main` all tests are green, so if anything fails in your PR, it's caused by new changes. You may run tests on bare `main` for comparison

Reply:
Thank @medikoo, The PR is now updated successfully.